src/index.ts

```diff
@@ -1,32 +1,19 @@
-import { getInput, info, setOutput } from "@actions/core";
-import { context, getOctokit } from "@actions/github";
-import { Context } from "@actions/github/lib/context";
-import { PullRequest } from "@octokit/webhooks-types";
+import { setFailed, setOutput } from "@actions/core";
 
-import { PullRequestApi } from "./github/pullRequest";
+import { fetchAllFellows } from "./fellows";
 import { generateCoreLogger } from "./util";
 
-const getRepo = (ctx: Context) => {
-  let repo = getInput("repo", { required: false });
-  if (!repo) {
-    repo = ctx.repo.repo;
-  }
+const logger = generateCoreLogger();
 
-  let owner = getInput("owner", { required: false });
-  if (!owner) {
-    owner = ctx.repo.owner;
-  }
-
-  return { repo, owner };
+const mapFellows = (fellows: Map<string, number>) => {
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const fellowsHandles = Array.from(fellows.keys());
+  setOutput("fellows-handles", JSON.stringify(fellowsHandles));
+  const fellowsMap = [...fellows].map(([name, rank]) => {
+    return { user: name, rank };
+  });
+  setOutput("fellows-map", JSON.stringify(fellowsMap));
+  setOutput("fellows", fellowsHandles.join(","));
```

Comment:
imo the action name `get-fellows-action` sounds to me that it gives just fellow's addresses and ranks maximum.. then when we map it to identity, the identity by itself also consists of many other fields besides Github and if we make it as generic action, it could potentially be re-used somewhere else besides review-bot? but the output assumes the only thing user would need is list of github usernames separated by comma.. 
i don't know if any other app besides review-bot would benefit from that
or if would benefit - then `fellows` sounds a bit too broad? or a bit vague in terms of expectations of what kind of data will be there - addresses? or addresses + rank, or full identity map etc

may be it makes sense to provide just one `fellows` array of objects which complete address,rank,identity data objects? and then consumer converts to any kind of form, whether array of GH handles or anything else

wdyt?

Reply:
I like that idea, I'll change the current `fellows` to be `fellows-github-handles`. Makes more sense